ubuntu_image/__main__.py

```diff
@@ -20,6 +20,44 @@
 PROGRAM = 'ubuntu-image'
 
 
+class SimpleHelpFormatter(argparse.HelpFormatter):
```

Comment:
Just an idea - would be nice to include a small docstring to mention what this formatter is doing.

Reply:
Sure.